docker-php/msmtprc

```diff
@@ -0,0 +1,12 @@
+## This file is used by docker for easy external SMTP email configuration
```

Comment:
This solution is here since by default the php docker library does not support `mail()` as per [this issue](https://github.com/docker-library/php/issues/135).